src/routes/anime/+layout.svelte

```diff
@@ -335,8 +336,9 @@
 					clip-rule="evenodd"
 					d="M24 0H0V24C0 10.7451 10.7452 0 24 0Z"
 				/>
-			</svg>
-			<div id="page" class="w-full overflow-y-scroll">
```

Comment:
dont remove it here, btw it was working fine on firefox
maybe you try it on ff?

https://coreproject-v4-ebfwbikcy-baseplateadmins-projects.vercel.app/anime/mal/1/episode/1